modules/tide_content_collection/src/Plugin/Field/FieldWidget/ContentCollectionConfigurationWidgetEnhanced.php

```diff
@@ -0,0 +1,639 @@
+      '@validation' => $schema_validation ? $this->t('Yes') : $this->t('No'),
+    ]);
+
+    $callToAction = $this->getSetting('callToAction');
+    $summary[] = $this->t('Call To Action Field: @status', [
+      '@status' => $callToAction ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $contentTypes = $this->getSetting('contentTypes');
+    $summary[] = $this->t('Content Types Field: @status', [
+      '@status' => $contentTypes ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $field_topic = $this->getSetting('field_topic');
+    $summary[] = $this->t('Content Topics Field: @status', [
+      '@status' => $field_topic ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $field_tags = $this->getSetting('field_tags');
+    $summary[] = $this->t('Content Tags Field: @status', [
+      '@status' => $field_tags ? $this->t('Enabled') : $this->t('Disabled'),
+    ]);
+
+    $entity_reference_fields = $this->getEntityReferenceFields();
+
+    if (!empty($entity_reference_fields)) {
+      foreach ($entity_reference_fields as $field_id => $field_label) {
+        $field_advanced = $this->getSetting($field_id);
+        $text = $this->t('Field: @status', [
+          '@status' => $field_advanced ? $this->t('Enabled') : $this->t('Disabled'),
+        ]);
+        $summary[] = $field_label['label'] . ' ' . $text;
+      }
+    }
+
+    return $summary;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function settingsForm(array $form, FormStateInterface $form_state) : array {
+    $element = parent::settingsForm($form, $form_state);
+
+    $element['schema_validation'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Validate against the JSON schema'),
+      '#default_value' => $this->getSetting('schema_validation'),
+    ];
+    $element['callToAction'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable/Disable Call To Action'),
+      '#default_value' => $this->getSetting('callToAction'),
+    ];
+    $element['contentTypes'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable/Disable Content Types'),
+      '#default_value' => $this->getSetting('contentTypes'),
+    ];
+    $element['contentFields']['field_topic'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable/Disable Content Topics Field'),
+      '#default_value' => $this->getSetting('field_topic'),
+    ];
+    $element['contentFields']['field_tags'] = [
+      '#type' => 'checkbox',
+      '#title' => $this->t('Enable/Disable Content Tags Field'),
+      '#default_value' => $this->getSetting('field_tags'),
+    ];
+
+    $entity_reference_fields = $this->getEntityReferenceFields();
+
+    if (!empty($entity_reference_fields)) {
+      foreach ($entity_reference_fields as $field_id => $field_label) {
+        $element['contentFields'][$field_id] = [
+          '#type' => 'checkbox',
+          '#title' => $this->t('Enable/Disable') . ' ' . $field_label['label'],
```

Comment:
@joshua-salsadigital it's not how the `t()` function should be used.
```suggestion
          '#title' => $this->t('Enable/Disable @field_label', ['@field_label' => $field_label['label']]),
```

Reply:
Made changes to code to follow correct way for translations.